### PR TITLE
gdiObjects.xBar version 2.0.2111.26

### PR DESCRIPTION
**Pull request by gdiObjects**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37388)

### CODE REVIEW

File: manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PackageLocale: en-US
+Publisher: gdiObjects
+PublisherUrl: https://www.gdiobjects.com/index.php
+PublisherSupportUrl: https://www.gdiobjects.com/helpdesk/
+PrivacyUrl: https://www.gdiobjects.com/privacy.php
+Author: gdiObjects
+PackageName: xBar
+PackageUrl: https://www.gdiobjects.com/Products/xBar/Index.php
+License: Copyright © 2008-2021 gdiObjects
+ShortDescription: The Fast & Easy Way to Find and Launch Software Apps, Open Documents, or Access Folders, and Drives in Windows. View System Details and Monitor your PC’s Performance with CPU, Memory, Temperature, and Battery Sensors.
```
Comment (Trenly) [inline manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.locale.en-US.yaml:15]:
```suggestion
ShortDescription: A Dockable Infobar and App Launcher for Windows
```

File: manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+PackageIdentifier: gdiObjects.xBar
+PackageVersion: 2.0.2111.26
+PackageLocale: en-US
+Publisher: gdiObjects
+PublisherUrl: https://www.gdiobjects.com/index.php
+PublisherSupportUrl: https://www.gdiobjects.com/helpdesk/
+PrivacyUrl: https://www.gdiobjects.com/privacy.php
+Author: gdiObjects
+PackageName: xBar
```
Comment (Trenly) [inline manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.locale.en-US.yaml:12]:
```suggestion
PackageName: xBar 2.0
```

File: manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: gdiObjects.xBar
+PackageVersion: 2.0.2111.26
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://www.gdiobjects.com/Files/XBInstall.exe
+  InstallerSha256: 3545A81910A18776E9D6BE6E6416E85BDA282CA843A0111AAA76F80B672080AE
```
Comment (Trenly) [inline manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.installer.yaml:10]:
```suggestion
  InstallerSha256: 3545A81910A18776E9D6BE6E6416E85BDA282CA843A0111AAA76F80B672080AE
  ProductCode: '{56E3DC02-59B7-4F96-AF45-F08C001F524E}_is1'
```

**Review by Trenly — CHANGES_REQUESTED** [review]
I don't know if this can actually be added to winget.

From gdiObjects ToS - https://www.gdiobjects.com/terms.php
> Except as expressly permitted by the copyright laws, no copying, storage, redistribution or publication of any Content is permitted without the express permission of gdiObjects or the owners of such Content or their authorized persons, if other than gdiObjects.

**Review by Trenly — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/37388#pullrequestreview-828485661

Just making sure the mods see ^ that comment

**Comment by gdiObjects** [comment]
> I don't know if this can actually be added to winget.
> 
> From gdiObjects ToS - https://www.gdiobjects.com/terms.php
> 
> > Except as expressly permitted by the copyright laws, no copying, storage, redistribution or publication of any Content is permitted without the express permission of gdiObjects or the owners of such Content or their authorized persons, if other than gdiObjects.

The ToS relates to the website (such as images).  Each software app includes a License Agreement (as part of the installer) and I also include the license agreement as a topic in the help file.

File: manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.locale.en-US.yaml (hunk 2 above)
Comment (gdiObjects) [reply manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.locale.en-US.yaml:12]: > 

please keep it as xBar (to be consistent with fotoXplorer's Package Name)

File: manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.installer.yaml (hunk 3 above)
Comment (gdiObjects) [reply manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.installer.yaml:10]: > 

I don't understand the differences between the InstallerSha256, they seem identical.  As to the ProductCode, that must be an internal reference.  I don't see it referenced in the *.yaml files.  I do have a question regarding the Architecture field.  xBar is a 64-bit  Windows application.  The installer is 32-bit.  Does the 'architecture' field relate to the app or the installer?

Comment (Trenly) [reply manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.installer.yaml:10]: It is the Product code that I was suggesting be added. Github is just weird about how it displays suggestions

Architecture refers to the application

File: manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.locale.en-US.yaml (hunk 2 above)
Comment (Trenly) [reply manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.locale.en-US.yaml:12]: This field has to match the value in the Add/Remove Programs table shown in control panel. In my control panel, it shows as `xBar 2.0`

**Comment by Trenly** [comment]
> > I don't know if this can actually be added to winget.
> > From gdiObjects ToS - https://www.gdiobjects.com/terms.php
> > > Except as expressly permitted by the copyright laws, no copying, storage, redistribution or publication of any Content is permitted without the express permission of gdiObjects or the owners of such Content or their authorized persons, if other than gdiObjects.
> 
> The ToS relates to the website (such as images). Each software app includes a License Agreement (as part of the installer) and I also include the license agreement as a topic in the help file.

Sounds good. Since the links to download the installer are technically website content, I just wanted to be sure that this type of redistribution wouldn’t be against your ToS

**Review by OfficialEsco — APPROVED** [review]
> I don't know if this can actually be added to winget.
> 
> From gdiObjects ToS - https://www.gdiobjects.com/terms.php
> 
> > Except as expressly permitted by the copyright laws, no copying, storage, redistribution or publication of any Content is permitted without the express permission of gdiObjects or the owners of such Content or their authorized persons, if other than gdiObjects.

Well, he IS the owner, so he chooses if winget can redistribute the package or not 🤷‍♂️

**Comment by Trenly** [comment]
> > I don't know if this can actually be added to winget.
> > From gdiObjects ToS - https://www.gdiobjects.com/terms.php
> > > Except as expressly permitted by the copyright laws, no copying, storage, redistribution or publication of any Content is permitted without the express permission of gdiObjects or the owners of such Content or their authorized persons, if other than gdiObjects.
> 
> Well, he IS the owner, so he chooses if winget can redistribute the package or not 🤷‍♂️

On my screen it just showed up as Steve Wehrmann, and not gdiObjects, which is why I was unsure

**Pull request merged**